espaloma/mm/nonbonded.py

```diff
@@ -119,21 +138,20 @@ def lj_9_6(x, sigma, epsilon):
     )
 
 
-#
-# def columb(x, q_prod, k_e=K_E):
-#     """ Columb interaction without cutoff.
-#
-#     Parameters
-#     ----------
-#     x : `torch.Tensor`, shape=`(batch_size, 1)` or `(batch_size, batch_size, 1)`
-#     q_prod : `torch.Tensor`,
-#         `shape=(batch_size, 1) or `(batch_size, batch_size, 1)`
-#
-#     Returns
-#     -------
-#     u : `torch.Tensor`,
-#         `shape=(batch_size, 1)` or `(batch_size, batch_size, 1)`
-#
-#
-#     """
-#     return k_e * x / q_prod
+def coulomb(x, q, k_e=K_E):
+    """ Columb interaction without cutoff.
+
+    Parameters
+    ----------
+    x : `torch.Tensor`, shape=`(batch_size, 1)` or `(batch_size, batch_size, 1)`
+    q_prod : `torch.Tensor`,
```

Comment:
Mismatch between parameter name and docstring.
Can we use an informative name like `chargeprod` here?
Be sure to document what units the charge product should be in.